superset/assets/src/visualizations/nvd3/NVD3Vis.js

```diff
@@ -260,6 +264,37 @@ function nvd3Vis(element, props) {
     return types.indexOf(vizType) >= 0;
   }
 
+  function findYAxisField(metrics, publihedColumns) {
```

Comment:
(nit) Typo in the second param name. `publihedColumns` should be `publishedColumns`

Reply:
done